integration_tests/mockApis/auth.ts

```diff
@@ -153,8 +153,8 @@ const stubUserRoles = () =>
 
 export default {
   getSignInUrl,
-  stubPing: (): Promise<[Response, Response]> => Promise.all([ping(), tokenVerification.stubPing()]),
```

Comment:
this was a bit odd since the health check was calling both auth and token verification anyway